src/popup/utils/constants.ts

```diff
@@ -8,21 +8,26 @@ import type {
 } from '../../types';
 import { i18n } from '../../store/plugins/languages';
 
+export { default as TIPPING_V1_ACI } from './contracts/tippingV1aci';
+export { default as TIPPING_V2_ACI } from './contracts/tippingV2aci';
+export { default as FUNGIBLE_TOKEN_CONTRACT_ACI } from './contracts/fungibleContractAci';
+export { default as PAIR_CONTRACT_ACI } from './contracts/pairContractAci';
+export { default as ZEIT_TOKEN_CONTRACT_ACI } from './contracts/zeitTokenAci';
+
 export const EXTENSION_HEIGHT = 600;
 export const MAGNITUDE = 18;
 export const SEED_LENGTH = 12;
 export const AETERNITY_CONTRACT_ID = 'aeternity';
 export const AETERNITY_SYMBOL = 'AE';
 
 export const TX_TYPE_MDW = {
```

Comment:
Should be completly removed, but this can be a separated issue, depending on what is current priority.

Reply:
@CedrikNikita You mean the ACIs shall be imported directly and not through constants?

Reply:
Was talking about `TX_TYPE_MDW` since now they are align with types from sdk.

Reply:
Done